src/bindings/c/include/openvino/c/ov_property.h

```diff
@@ -0,0 +1,150 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+/**
+ * @brief This is header file for ov_property C API.
+ * A header for advanced hardware specific properties for OpenVINO runtime devices.
+ * To use in set_property, compile_model, import_model, get_property methods.
+ * @file ov_property.h
+ */
+
+#pragma once
+
+#include "openvino/c/ov_common.h"
+
+typedef struct ov_property ov_property_t;
+
+/**
+ * @enum ov_performance_mode_e
+ * @brief Enum to define possible performance mode hints
+ * @brief This represents OpenVINO 2.0 ov::hint::PerformanceMode entity.
+ *
+ */
+typedef enum {
+    UNDEFINED_MODE = -1,        //!<  Undefined value, performance setting may vary from device to device
+    LATENCY = 1,                //!<  Optimize for latency
+    THROUGHPUT = 2,             //!<  Optimize for throughput
+    CUMULATIVE_THROUGHPUT = 3,  //!< Optimize for cumulative throughput
+} ov_performance_mode_e;
+
+/**
+ * @enum ov_affinity_e
+ * @brief Enum to define possible affinity patterns
```

Comment:
can we provide doxygen example how to set this enum as a property? It's not obvious for me..

Reply:
sure, we should have doc for C api usage guideline